arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -1089,6 +1101,12 @@ ::get_configuration() const
     "standard for each frame with the current value of every existing tag. "
     "Otherwise, will report packets as they appear in the source video." );
 
+  config->set_value(
+    "unknown_stream_behavior", d->unknown_stream_behavior,
+    "Set to 'klv' to treat unknown streams as KLV (default). "
```

Comment:
This should probably default to ignore? Not sure but I figure default should err on the side of caution